### PR TITLE
Docker Desktop 4.3.2 release notes

### PR DESCRIPTION
**Pull request by lorenrh**
Signed-off-by: Lorena Rangel <lorena.rangel@docker.com>

### Proposed changes

- Update release notes for Docker Desktop 4.3.2
- Added the link to the blogpost as in https://github.com/docker/docker.github.io/pull/13968

### CODE REVIEW

File: desktop/mac/release-notes/index.md [hunk 1]
```diff
@@ -31,6 +31,29 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
 
+[docker scan v0.14.0](https://github.com/docker/scan-cli-plugin/releases/tag/v0.14.0){: target="_blank" rel="noopener" class="_"}
+
+### Security
+
+**Log4j 2 CVE-2021-44228**: We have updated the `docker scan` CLI plugin for you.
+Older versions of `docker scan` in Docker Desktop 4.3.0 and earlier versions are
+not able to detect [Log4j 2
+CVE-2021-44228](https://nvd.nist.gov/vuln/detail/CVE-2021-44228){:
+target="_blank" rel="noopener" class="_"}.
```
Comment (stephanierifai) [inline desktop/mac/release-notes/index.md:42]:
```suggestion
target="_blank" rel="noopener" class="_"} nor [Log4j 2
CVE-2021-45046](https://nvd.nist.gov/vuln/detail/CVE-2021-45046).
```

File: desktop/windows/release-notes/index.md [hunk 2]
```diff
@@ -30,6 +30,28 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
 
+[docker scan v0.14.0](https://github.com/docker/scan-cli-plugin/releases/tag/v0.14.0){: target="_blank" rel="noopener" class="_"}
+
+### Security
+
+**Log4j 2 CVE-2021-44228**: We have updated the `docker scan` CLI plugin for you.
+Older versions of `docker scan` in Docker Desktop 4.3.0 and earlier versions are
+not able to detect [Log4j 2
+CVE-2021-44228](https://nvd.nist.gov/vuln/detail/CVE-2021-44228){:
+target="_blank" rel="noopener" class="_"}.
```
Comment (stephanierifai) [inline desktop/windows/release-notes/index.md:41]:
```suggestion
target="_blank" rel="noopener" class="_"} nor [Log4j 2
CVE-2021-45046](https://nvd.nist.gov/vuln/detail/CVE-2021-45046).
``

File: desktop/mac/release-notes/index.md [hunk 3]
```diff
@@ -31,6 +31,29 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 ### Upgrades
 
+[docker scan v0.14.0](https://github.com/docker/scan-cli-plugin/releases/tag/v0.14.0){: target="_blank" rel="noopener" class="_"}
+
+### Security
+
+**Log4j 2 CVE-2021-44228**: We have updated the `docker scan` CLI plugin for you.
```
Comment (stephen-turner) [inline desktop/mac/release-notes/index.md:38]: This is a bit informal for a release note. Delete "for you".

File: desktop/mac/release-notes/index.md [hunk 4]
```diff
@@ -31,6 +31,29 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+Older versions of `docker scan` in Docker Desktop 4.3.0 and earlier versions are
```
Comment (stephen-turner) [inline desktop/mac/release-notes/index.md:39]: Turn round into a positive: This new version of `docker scan` is able to detect...

**Review by stephen-turner — CHANGES_REQUESTED** [review]

**Review by jerae-duffin — APPROVED** [review]
LGTM!

**Review by stephanierifai — APPROVED** [review]
LGTM!

**Review by StefanScherer — APPROVED** [review]
LGTM

**Pull request merged**